external-providers/generic-external-provider/pkg/generic/dependency.go

```diff
@@ -0,0 +1,46 @@
+package generic
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os/exec"
+
+	"github.com/konveyor/analyzer-lsp/provider"
+	"go.lsp.dev/uri"
+)
+
+func (g *genericServiceClient) GetDependencies() (map[uri.URI][]*provider.Dep, error) {
+	buf := bytes.Buffer{}
+	cmdStr, isString := g.config.ProviderSpecificConfig["dependencyProviderPath"].(string)
+	if !isString {
+		return nil, fmt.Errorf("dependency provider path is not a string")
+	}
+	cmd := exec.Command(cmdStr)
+	cmd.Dir = g.config.Location
+	cmd.Stdout = &buf
```

Comment:
I think you'd typically capture output more like this:

 ```
 out, err := cmd.Output()
```

rather than overwriting the stdout buffer

Reply:
aah yes, ill do that

Reply:
@fabianvf should i also do the same when I take the output from `go mod graph` ?